tests/time_tests/c++/ie_pipeline.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
```

Comment:
in general - update namings with CamelCase formatting

Reply:
Done for classes and functions, ignored for variables (but it still compatible with IE formatting)